packages/pwa-kit-react-sdk/setup-jest.js

```diff
@@ -38,7 +38,7 @@ jest.mock('pwa-kit-runtime/utils/ssr-config', () => {
                         path: 'api'
                     },
                     {
-                        host: 'zzrf-001.sandbox.us03.dx.commercecloud.salesforce.com',
+                        host: 'https://zzrf-001.dx.commercecloud.salesforce.com',
```

Comment:
Is this correct? I think typically `URL` includes the protocol (http[s]) but `host` does not 🤔

Reply:
Oh yes. I'll remove the protocol from host values. Thanks for pointing that out.